StreamChatUITestsAppUITests/MockResponses/ws_message_read.json

```diff
@@ -0,0 +1,21 @@
+
+{
+  "type": "message.read",
+  "cid": "messaging:general",
+  "channel_id": "general",
+  "channel_type": "messaging",
+  "user": {
+    "id": "leia_organa",
+    "role": "user",
+    "created_at": "2020-12-07T11:37:33.78879Z",
+    "updated_at": "2022-04-20T16:07:59.40095Z",
+    "last_active": "2022-05-02T11:47:15.218330756Z",
+    "banned": false,
+    "online": true,
+    "AdditionalProperties": {},
+    "name": "Leia Organa",
+    "image": "https://vignette.wikia.nocookie.net/starwars/images/f/fc/Leia_Organa_TLJ.png",
+    "birthland": "Polis Massa"
+  },
+  "created_at": "2022-05-02T11:57:25.372161423Z"
+}
```

Comment:
I believe we can use `ws_events.json` instead of creating a new file here, can't we? Tbh, I think we're already doing it 🤔

Reply:
removed, good catch ;)